docs/source/technical_tutorials/working_with_r/access_aws_maap.ipynb

```diff
@@ -0,0 +1,1588 @@
+{
```

Comment:
<p>Are we missing a <code>library("paws")</code> line in the preceding cell? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/471/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>